services/static-webserver/client/source/class/osparc/FlashMessenger.js

```diff
@@ -111,9 +111,16 @@ qx.Class.define("osparc.FlashMessenger", {
       }
       this.__displayedMessagesCount++;
 
+      const message = flashMessage.getMessage();
+      if (!message) {
+        // If message is "", null or undefined, do not show it
+        this.removeMessage(flashMessage);
+        return;
+      }
+
```

Comment:
Why don't you do this before adding it to the container?

Reply:
reverting